modules/house/add.vue

```diff
@@ -1,68 +1,77 @@
+			</el-col>
+		</el-row>
+		<div v-if="form.houseFormat!==userInfo.houseTypes.ENTIRE[0]">
+			<el-row :gutter="20">
+				<el-col :span="8">
 					<el-input v-model="building" @blur="changeBuilding">
 						<template slot="prepend">序号</template>
 						<template slot="append">幢</template>
 					</el-input>
-					</el-col>
-					<el-col :span="8">
+				</el-col>
+				<el-col :span="8">
 					<el-input v-model="unit" @blur="changeUnit">
 						<template slot="append" style="width:53px">单元</template>
 					</el-input>
-					</el-col>
-					<el-col :span="8">
+				</el-col>
+				<el-col :span="8">
 					<el-input v-model="form.roomNumber">
 						<template slot="append">室</template>
 					</el-input>
-					</el-col>
-				</el-row>
-			</div>
-			<room-layout v-model="form.layout" :rentType="form.houseFormat"></room-layout>
-			<building-floor v-if="form.houseFormat==userInfo.houseTypes.ENTIRE[0]" :data="Entire"></building-floor>
-			<div v-if="form.houseFormat!==userInfo.houseTypes.ENTIRE[0]">
-				<el-row :gutter="20">
-					<el-col :span="8" class="roomInfo">
-						<el-input v-model.number="form.layout.roomArea" auto-complete="off" placeholder="面积">
-							<template slot="prepend">信息</template>
-							<template slot="append">㎡</template>
-						</el-input>
-					</el-col>
-					<el-col :span="8" class="roomInfo">
-						<el-input v-model.number="form.currentFloor" auto-complete="off" placeholder="层高">
-							<template slot="append">层</template>
-						</el-input>
-					</el-col>
-					<el-col :span="8" class="roomInfo">
-						<el-input v-model.number="form.totalFloor" auto-complete="off" placeholder="总层数">
-							<template slot="append">层</template>
-						</el-input>
-					</el-col>
-				</el-row>
-			</div>
-			<h3 style="margin-top:30px">房源配置</h3>
+				</el-col>
+			</el-row>
+		</div>
+		<room-layout v-model="form.layout" :rentType="form.houseFormat"></room-layout>
+		<building-floor v-if="form.houseFormat==userInfo.houseTypes.ENTIRE[0]" :data="Entire"></building-floor>
+		<div v-if="form.houseFormat!==userInfo.houseTypes.ENTIRE[0]">
+			<el-row :gutter="20">
+				<el-col :span="8" class="roomInfo">
+					<el-input v-model.number="form.layout.roomArea" auto-complete="off" placeholder="面积">
+						<template slot="prepend">信息</template>
+						<template slot="append">㎡</template>
+					</el-input>
+				</el-col>
+				<el-col :span="8" class="roomInfo">
+					<el-input v-model.number="form.currentFloor" auto-complete="off" placeholder="层高">
+						<template slot="append">层</template>
+					</el-input>
+				</el-col>
+				<el-col :span="8" class="roomInfo">
+					<el-input v-model.number="form.totalFloor" auto-complete="off" placeholder="总层数">
+						<template slot="append">层</template>
+					</el-input>
+				</el-col>
+			</el-row>
+      <el-row :gutter=20>
+        <el-col :span=8 class=roomInfo>
+          <el-input v-model.number="form.electricPrice" auto-complete=off type=number step=any>
```

Comment:
I would suggest to use `el-input-number` for price input, you can find some references in other files